pxtlib/github.ts

```diff
@@ -557,7 +557,18 @@ namespace pxt.github {
             return Promise.resolve<CachedPackage>(undefined);
         }
 
-        return db.loadPackageAsync(p.fullName, p.tag);
+        return db.loadPackageAsync(p.fullName, p.tag)
+            .then(cached => {
+                const dv = upgradedDisablesVariants(config, repoWithTag)
+                if (dv) {
+                    const cfg = Package.parseAndValidConfig(cached.files[pxt.CONFIG_NAME])
+                    if (cfg) {
+                        cfg.disablesVariants = dv
+                        cached.files[pxt.CONFIG_NAME] = JSON.stringify(cfg, null, 4)
```

Comment:
We also have an api to strongly the pxt.json . I think stringingConfig or something. It makes sure that fields are in same order etc